settings.gradle

```diff
@@ -10,6 +10,7 @@ include ':paparazzi-agent'
 include ':paparazzi-gradle-plugin'
 
 include ':sample'
+include ':sample-module'
```

Comment:
i prefer not to add a new sample module for this, especially with the discussed plan to overhaul the sample soon.  can we revert that part and maybe also update the existing plugin test?